security/advisories/index.html

```diff
@@ -23,6 +23,11 @@
                     </tr>
                 </thead>
                 <tbody>
+                     <tr>
+                        <td>August 13, 2025</td>
+                        <td><a href="{{ site.baseurl }}/security/advisories/2025-54791">CVE-2025-54791 ("OMERO.web displays unnecessary user information when requesting to reset the password")</a></td>
```

Comment:
Only minor concern is that the parenthesis title is fairly long. Is there a way to shorten it?

Reply:
It's the title we gave it in GitHub - happy to shorten it if they don't need to match

Reply:
I think it's more important to update the website and we can review separately whether both titles should be shortened. Maybe it's a display question i.e. the CVE title should be in a separate column from the CVE ID